package.json

```diff
@@ -17,6 +17,7 @@
     "@pothos/core": "3.24.0",
     "@pothos/plugin-relay": "3.32.0",
     "@pothos/plugin-simple-objects": "^3.7.0",
+    "@uidotdev/usehooks": "^2.4.1",
```

Comment:
I've been seeing this usehooks.com referenced for the past ~year so it's cool to have a solid use-case for importing it